src/mame/drivers/tetrisp2.cpp

```diff
@@ -1129,6 +1142,12 @@ INPUT_PORTS_END
 ***************************************************************************/
 
 static INPUT_PORTS_START( stepstag )
+	PORT_START("VOLUME")	// $a30000.w
+	PORT_DIPNAME( 0x00ff, 0x0000, "Sound Volume")	// Potentiometer output sampled by 8-bit A/D, then read by main CPU
+	PORT_DIPSETTING(      0x0000, "max")			// TODO, 256 steps
+	PORT_DIPSETTING(      0x007f, "mid")
+	PORT_DIPSETTING(      0x00ff, "min")
```

Comment:
You can use an “adjuster” rather than DIP switches as an easier way to get a value from a range like this.

Reply:
Was indeed wondering if we have one or not. -> [`PORT_ADJUSTER`](https://github.com/mamedev/mame/blob/ee1e4f9683a4953cb9d88f9256017fcbc38e3144/src/mame/drivers/starfire.cpp#L293)